File: FrizerskiSalon.Mobile/frizerski_salon_app/lib/model/StavkeNarudzbe.dart
class StavkeNarudzbe{
  int? stavkeNarudzbeId;
  int? proizvodId;
  int? narudzbaId;
  int? kolicina;

  StavkeNarudzbe({this.stavkeNarudzbeId, this.proizvodId, this.narudzbaId, this.kolicina});

  factory StavkeNarudzbe.fromJson(Map<String, dynamic> json) {
    return StavkeNarudzbe(
      stavkeNarudzbeId:int.parse(json["stavkeNarudzbeId"].toString()),
      proizvodId: int.parse(json["proizvodId"].toString()),
      narudzbaId: int.parse(json["narudzbaId"].toString()),
      kolicina: json["kolicina"]
    );
  }

  Map<String, dynamic> toJson() => {
    "stavkeNarudzbeId":stavkeNarudzbeId,
    "proizvodId":proizvodId,
    "narudzbaId":narudzbaId,
    "kolicina":kolicina
  };
}

File: FrizerskiSalon.Modal/StavkeNarudzbe.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FrizerskiSalon.Modal
{
    public class StavkeNarudzbe
    {
        public int StavkeNarudzbeId { get; set; }
        public int? NarudzbaId { get; set; }
        public Narudzba Narudzba { get; set; }
        public int? ProizvodId { get; set; }
        public Proizvod Proizvod { get; set; }
        public int? Kolicina { get; set; }
    }
}
